enzyme/Enzyme/EnzymeLogic.cpp

```diff
@@ -2069,9 +2069,9 @@ void createTerminator(DiffeGradientUtils *gutils,
 
     auto retVal = inst->getOperand(0);
 
```

Comment:
Why is isConstantValue insufficient here and we need to also explicitly check the diffe_type?

Reply:
Yes, this should become unnecessary once I've figured out the activity of return check